examples/platforms/nrf52840/system.c

```diff
@@ -84,6 +85,9 @@ void otSysInit(int argc, char *argv[])
     nrf5SpiSlaveInit();
 #endif
     nrf5MiscInit();
+#if ENABLE_FEM
+    PlatformFemSetConfigParams(&PLATFORM_FEM_DEFAULT_CONFIG);
```

Comment:
I would call `PlatformFemSetConfigParams` function after `nrf5RadioInit()`. In fact i would prefer to call it at the end like that:

```
    nrf5RadioInit();
    nrf5TempInit();

#if PLATFORM_FEM_SET_DEFAULT_CONFIG
    PlatformFemSetConfigParams(&PLATFORM_FEM_DEFAULT_CONFIG);
#endif

    gPlatformPseudoResetWasRequested = false;
```